src/pages/library/Datasets.js

```diff
@@ -422,25 +422,27 @@ const topMed = () => h(Participant, {
   }, ['Browse Data'])
 ])
 
-const rareX = () => h(Participant, {
-  logo: { src: rareXLogo, alt: 'RareX logo' },
-  title: 'RARE-X Data Analysis Platform',
-  description: h(Fragment, [
-    h(Link, { href: 'https://rare-x.org', ...Utils.newTabLinkProps }, 'The RARE-X Data Analysis Platform'),
-    ` is a federated data repository of rare disease patient health data.
+const rareX = () => {
+  const linkToRareXPublicWorkspaces = `${Nav.getLink('workspaces')}/?tab=public&tagsFilter%5B%5D=rare-x`
```

Comment:
Since this is only used once, why store it in a variable instead of inlining it?

Reply:
I inlined it first and then thought the meaning of the URL piece we're appending may not be obvious so I indirectly commented by pulling it. If you feel that's unnecessary, I can revert.

Reply:
I don't think it matters much here, but I think a comment may have served that purpose better than a variable name.

Reply:
Changed in [c9f5043](https://github.com/DataBiosphere/terra-ui/pull/3374/commits/c9f5043b658cb5ddc8d7a001ec4c9fc6a113c328).